src/main/java/frc/robot/Constants.java

```diff
@@ -19,4 +19,5 @@
     public static int MAX_COLOR_ROTATIONS = 3;
     public static String SPIN_COLOR = "";
     public static boolean IS_SPIN_COMPLETE = false;
+    public static boolean IS_CAPTURE_COMPLETE = true;
```

Comment:
Is this used anywhere? I couldn't find any references.

Reply:
Delete